### DISCUSSION TITLE
App restart unloads maps / poi

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#262] [Q&A] [answered]
Originally reported in https://github.com/devemux86/cruiser/discussions/256#discussioncomment-6593930.

**Comment by devemux86** [comment]
After some back and forth loading maps / poi and restarting the app  I was able to reproduce it.

We can thank the new version of the Android build environment that was recently released…

I will fix it in a new Cruiser version that will be released in the next few hours.

**Reply by 1maxi2** [reply]
Sicher nicht einfach bei den sich ständig Veränderungen eine SW zu entwickeln ( weiter zu entwickeln)
Danke für deinen Einsatz und die immer schnellen Reaktionen/ Antworten bei Problemen !

**Comment by devemux86** [comment]
I found the fix for the Android issue, it will be included in the next Cruiser update.

Will be released as soon as possible! :slightly_smiling_face:

**Reply by Leeb64** [reply]
Android developers...all specialists...

**Reply by devemux86** [reply]
Also important is how much time we have to spend on such irrelevant things,
instead of developing new features in applications. It's a really sad situation...

**Comment by devemux86** [comment, marked as answer]
[Cruiser 3.0.28](https://play.google.com/store/apps/details?id=gr.talent.cruiser) with the fix was released on Google Play.

Sorry for the inconvenience and thanks everyone for the report!
